Portfolio.html

```diff
@@ -3,12 +3,86 @@
 <head>
     <meta charset="UTF-8">
     <link href="https://fonts.googleapis.com/css?family=Quicksand" rel="stylesheet">
-    <link rel="stylesheet" type="text/css" href="Portfolio.css">
-    <title>Rob Reeves Portfolio</title>
+    <link href="https://fonts.googleapis.com/css?family=Fanwood+Text" rel="stylesheet">
+    <link rel="stylesheet" type="text/css" href=":css/normalize.css">
+    <link rel="stylesheet" type="text/css" href=":css/portfolio.css">
+    <link rel="stylesheet" type="text/css" href=":css/media-query.css">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Rob Reeves | Portfolio</title>
 </head>
-<body>
-
-
 
+<body>
+    <header class="main-nav">
+        <div class="nav-button-left">
+            <a class="nav-text" href="#C1">ABOUT ME</a>
+            <a class="nav-text" href="#C3">CONTACT</a>
+        </div>
+        <div class="nav-button-right">
+            <a class="nav-text" href="#C2">THREE PEDALS ONLY</a>
+        </div>
+    </header>
+        <div class="main-header-wrapper">
+            <div class="main-header-centre">
+                <h1>ROB REEVES</h1>
+                    <div class="sub-heading-wrapper">
+                        <div class="sub-heading">
+                            <h3>JUNIOR WEB DEVELOPER</h3>
+                        </div>
+                    </div>
+                    <div class="sub-heading-wrapper2">
+                        <p class="sub-heading2">There are lots of interesting things about me on this website. Please take your time and enjoy your stay. If you have any queries about anything then please don't hesitate to drop me a line.</p>
+                    </div>
+            </div>
+        </div>
+    <section class="main-content" id="C1">
+        <div class="main-content-wrapper">
+            <div class="h2-sub-heading">
```

Comment:
classname with a tag in it is bad practise and not semantic. sub-heading will do.

Reply:
fixed.